fsharp-backend/src/LibExecution/StdLib/LibList.fs

```diff
@@ -1085,38 +1098,33 @@ let fns : List<BuiltInFn> =
 //     ; sqlSpec = NotYetImplementedTODO
 //     ; previewable = Pure
 //     ; deprecated = NotDeprecated }
-//   ; { name = fn "List" "getAt" 0
-//     ; parameters = [Param.make "list" TList ""; Param.make "index" TInt ""]
-//     ; returnType = TOption
-//     ; description =
-//         "Returns `Just value` at `index` in `list` if `index` is less than the length of the list. Otherwise returns `Nothing`."
-//     ; fn =
-//           (function
-//           | _, [DList l; DInt index] ->
-//               List.nth l (Dint.to_int_exn index)
-//               |> Option.map (fun a -> DOption (OptJust a))
-//               |> Option.value (DOption OptNothing)
-//           | _ ->
-//               incorrectArgs ())
-//     ; sqlSpec = NotYetImplementedTODO
-//     ; previewable = Pure
-//     ; deprecated = ReplacedBy(fn "" "" 0) }
-//   ; { name = fn "List" "getAt" 1
-//     ; parameters = [Param.make "list" TList ""; Param.make "index" TInt ""]
-//     ; returnType = TOption
-//     ; description =
-//         "Returns `Just value` at `index` in `list` if `index` is less than the length of the list otherwise returns `Nothing`."
-//     ; fn =
-//           (function
-//           | _, [DList l; DInt index] ->
-//               List.nth l (Dint.to_int_exn index)
-//               |> Option.map (fun a -> Dval.to_opt_just a)
-//               |> Option.value (DOption OptNothing)
-//           | _ ->
-//               incorrectArgs ())
-//     ; sqlSpec = NotYetImplementedTODO
-//     ; previewable = Pure
-//     ; deprecated = NotDeprecated }
+    { name = fn "List" "getAt" 0
+      parameters = [Param.make "list" (TList varA) ""; Param.make "index" TInt ""]
+      returnType = TOption varA
+      description =
+        "Returns `Just value` at `index` in `list` if `index` is less than the length of the list. Otherwise returns `Nothing`."
+      fn =
+        (function
+        | _, [ DList l; DInt index ] ->
+            (match List.tryItem (int index) l with
+             | Some d -> Value(Dval.optionJust d)
```

Comment:
I believe this should use `(DOption (Some d))` instead of `optionJust` to match the OCaml implementation (the difference being that it doesn't propagate fakevals.

Reply:
Good point, I will change it.